### ISSUE TITLE
Fix activity indicator's accessibility violation

### ISSUE THREAD

**Issue by kittsville** [kittsville/Hypervote#7]
The Activity Indicator currently uses opacity to indicate whether it should be displayed or not. This is not compatible with screen readers as opacity has no effect on them.
